integration-tests/README.md

```diff
@@ -53,13 +53,13 @@ You can use this flag to test in a remote machine too.
 With the --filter flag you can select the tests to run. For instance you can
 pass MyTestSuite, MyTestSuite.FirstCustomTest or MyTestSuite.*CustomTest:
```

Comment:
Making a new branch to backtick the README. Thanks for the review, I'll land this.